mdpow/templates/bar_opls.mdp

```diff
@@ -1,7 +1,7 @@
-; Gromacs mdp file
-; Template for MD Hydration FEP with OPLS-AA (part of POW)
+; GROMACS mdp file
+; Template for MD Hydration FEP with OPLS-AA (part of MDPOW)
```

Comment:
Same comment as above: OPLS-AA .mdp files are also used for AMBER/GAFF, in which case the description will not be correct. I would put something like "OPLS-AA or AMBER".

Reply:
I would also put "Solvation" instead of "Hydration".

Reply:
Maybe also change the name of the file into "bar_opls_amber.mdp", which is more clear.

Reply:
made amber version